src/evaluate.py

```diff
@@ -140,18 +141,21 @@ def write_preds(tasks: Iterable[tasks_module.Task], all_preds, pred_dir, split_n
                 strict_glue_format and task.name in tasks_module.ALL_GLUE_TASKS)
             _write_glue_preds(task.name, preds_df, pred_dir, split_name,
                               strict_glue_format=strict)
-            log.info("Task '%s': Wrote predictions to %s", task.name, pred_dir)
         elif isinstance(task, EdgeProbingTask):
             # Edge probing tasks, have structured output.
             _write_edge_preds(task, preds_df, pred_dir, split_name)
             log.info("Task '%s': Wrote predictions to %s", task.name, pred_dir)
         elif isinstance(task, CommitmentTask):
             _write_commitment_preds(task, preds_df, pred_dir, split_name,
                                     strict_glue_format=strict_glue_format)
+        elif isinstance(task, COPATask):
+            _write_copa_preds(task, preds_df, pred_dir, split_name,
+                              strict_glue_format=strict_glue_format)
         else:
             log.warning("Task '%s' not supported by write_preds().",
                         task.name)
             continue
+        log.info("Task '%s': Wrote predictions to %s", task.name, pred_dir)
```

Comment:
It looks like this'll appear twice now for Edge Probing, no?

Reply:
fixed